conn.go

```
@@ -266,15 +274,50 @@ func (c *Conn) setSchemaV2(s bool) {
 	c.isSchemaV2 = s
 }
 
+// systemRequestState is the immutable pair of timeouts a driver-issued system
+// query is sent with: the client-side deadline, and the ScyllaDB-only
+// " USING TIMEOUT ...ms" clause pre-rendered from it (empty when the clause does
+// not apply). The two are published as one snapshot so a reader can never pair
+// one with a stale version of the other - which would ask the server to abort a
+// system query on a deadline the client is not waiting on, or vice versa.
+type systemRequestState struct {
```

golangci-lint (2.12.2)

[error] fieldalignment: struct with 16 pointer bytes could be 8

(govet)